package.json

```diff
@@ -165,7 +165,8 @@
   },
   "jest": {
     "collectCoverageFrom": [
-      "!src/**/*.stories.{jsx,tsx}"
+      "!src/**/*.stories.{jsx,tsx}",
+      "!src/**/__snapshots__/*.test.tsx.snap"
```

Comment:
Thanks for doing that- I remember you were saying snapshots skew the reports, and didn't think of removing them from coverage as an option